docs/how-to-guides/manage-user-privileges.md

```diff
@@ -74,7 +74,7 @@ We could naturally cluster those privileges into groups:
   * Privilege to try freeing memory (`FREE_MEMORY`)
   * Privilege to use trigger queries (`TRIGGER`)
   * Privilege to use stream queries (`STREAM`)
-  * Privilege to configure Memgraph during runtime (`CONFIG`)
+  * Privilege to configure Memgraph during runtime and to attain the configuration of the given Memgraph instance(`CONFIG`)
```

Comment:
```suggestion
  * Privilege to configure Memgraph during runtime and to access the configuration of the given Memgraph instance(`CONFIG`)
```

Reply:
to access configuration to me implies I can do something with it (like change it) :)